backend/src/main/java/com/airing/backend/image/controller/ImageController.java

```diff
@@ -0,0 +1,39 @@
+package com.airing.backend.image.controller;
+
+import com.airing.backend.auth.jwt.JwtProvider;
+import com.airing.backend.image.dto.PresignedUrlResponse;
+import com.airing.backend.image.service.ImageService;
+import com.airing.backend.user.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/images")
+@RequiredArgsConstructor
+public class ImageController {
+
+    private final ImageService imageService;
+    private final JwtProvider jwtProvider;
+    private final UserRepository userRepository;
+
+    @PostMapping("/presigned-url")
+    public ResponseEntity<List<PresignedUrlResponse>> getPresignedUrls(
+            @RequestBody List<String> filenames,
```

Comment:
Request body에는 파일 확장자 리스트인 `fileTypes`를 받아야 합니다.
```suggestion
            @RequestBody List<String> fileTypes,
```